core/app/models/mno_enterprise/tenant.rb

```diff
@@ -1,6 +1,55 @@
 module MnoEnterprise
   class Tenant < BaseResource
+    # == Attributes ===========================================================
     property :created_at, type: :time
     property :updated_at, type: :time
+#   property :frontend_config
```

Comment:
Remove? Note that if you do not specify the property you may receive a "NoMethod" error when calling the property when nil.